integrations/sonarqube/main.py

```diff
@@ -33,6 +33,13 @@ def organization_key_missing_for_onpremise() -> bool:
     )
 
 
+def organization_key_missing_for_sonarcloud() -> bool:
+    return (
+        not ocean.integration_config.get("sonar_organization_id")
+        and not is_onpremise_deployment()
+    )
```

Comment:
I feel like both methods are quite redundant, as you don't re-use it anywhere else.